conans/client/tools/oss.py

```diff
@@ -24,12 +25,38 @@ def args_to_string(args):
 
 def cpu_count(output=None):
     output = default_output(output, 'conans.client.tools.oss.cpu_count')
+
+    def is_running_in_docker():
+        try:
+            cgroup = "/proc/self/cgroup"
+            return (os.path.exists('/.dockerenv') or
+                    (os.path.isfile(cgroup) and any('docker' in line for line in open(cgroup))))
+        except:
+            return False
+
+    def docker_cpu_count():
+        base_cpus = "/sys/fs/cgroup/cpu"
+        try:
+            with open(os.path.join(base_cpus, "cpu.cfs_quota_us"), 'r') as quota_fd:
```

Comment:
also, I think you may use `tools.load` here, code will be shorter

Reply:
thanks! I was trying to remember which Conan helper could read a file.

Reply:
done